config/packages/security.yaml

```diff
@@ -26,7 +26,7 @@ security:
 
         api_key:
             anonymous: ~
-            pattern: '^%app.route_prefix%/v1/bulk'
+            pattern: '^%app.route_prefix%/v1/(bulk|line-items)'
```

Comment:
Just do not forget to notify the applications who are using this endpoint that it was changed :)